metron-analytics/metron-profiler-storm/pom.xml

```diff
@@ -130,7 +130,7 @@
         </dependency>
         <dependency>
             <groupId>org.apache.metron</groupId>
-            <artifactId>metron-hbase</artifactId>
+            <artifactId>metron-hbase-storm</artifactId>
```

Comment:
All of the classes in `metron-hbase-storm` like `HBaseBolt`, `HBaseMapper`, `BatchHelper` are **only** used by the Profiler.  These classes will never be reused by anything else at this point.  Would it make sense to just move these to `metron-profiler-storm` and remove the new `metron-hbase-storm` project?

Reply:
Done